packages/styles/scss/patterns/blocks/leadspace-block/_leadspace-block.scss

```diff
@@ -0,0 +1,89 @@
+@import '../../../globals/imports';
+@import '../../../globals/utils/content-width';
+@import '../../../globals/utils/hang';
+
+
+@mixin leadspace-block {
+
+  .#{$prefix}--leadspace-block {
+    padding-left: $spacing-05;
+    padding-right: $spacing-05;
+
+    &__title {
+      width: 90%;
+      color: $text-01;
+      padding-top: $layout-05;
+      @include carbon--type-style('expressive-heading-06', true);
+
+      @include carbon--breakpoint('lg') {
+          padding-top: $layout-05;
+      }
+
+      @include carbon--breakpoint('md') {
+          padding-top: $layout-03;
+      }
+
+      @include carbon--breakpoint('sm') {
+          padding-top: $layout-03;
+      }
+    }
+
+    &__heading {
+      @include carbon--type-style('expressive-heading-05', true);
+    }
+
+    .#{$prefix}--content-block {
+      padding-top: 0;
+      padding-bottom: $layout-05;
+
+      @include carbon--breakpoint('sm') {
+        padding-bottom: $layout-01;
+      }
+      @include carbon--breakpoint('md') {
+        padding-bottom: $layout-06;
+      }
+
+      .#{$prefix}--image-with-caption {
+        margin-bottom: 0;
+      }
+
+      .#{$prefix}--link-list {
+        padding-top: $spacing-09;
+        padding-bottom: $spacing-09;
+      }
+    }
+  }
+
+  .#{$prefix}--leadspace-block__media {
+    padding-top: $spacing-05;
+
+    .#{$prefix}--image-with-caption {
+      margin-top: 0;
+    }
+  }
+
+  .#{$prefix}--buttongroup {
+    padding-top: 0;
+
+    .#{$prefix}--buttongroup-item {
+      margin-top: 0;
+    }
+
+    .#{$prefix}--btn {
+      width: 60%;
+    }
+  }
+  .#{$prefix}--buttongroup-item {
+    margin-top: 0;
+    padding-top: 0;
+  }
+
+  .#{$prefix}--hr {
+    border-color: $ui-04;
+    margin-top: 0;
+  }
```

Comment:
The hr was affected globally. Can you scope this to this pattern?

Reply:
yeah, I will change it @jeffchew

Reply:
moved the style @jeffchew